scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -1188,11 +1188,22 @@ <h6 class="card-subtitle mb-2 text-muted"><span class="badge badge-warning">Part
               </td>
             </tr>
           {% endfor %}
-        {% endif %}
-      </tbody>
+        </tbody>
+      {% elif variant.genes %} <!-- more than 5 genes in this SV -->
+        <thead>
+          <th>{{ variant.genes|length }} affected gene(s)</th>
+        </thead>
+        <tbody>
+          <td>
+            {% for gene in variant.genes|sort(attribute='common.hgnc_symbol') %} <!-- Display all genes sorted by symbol -->
+              <a href="{{ url_for('genes.gene', hgnc_id=gene.hgnc_id) }}">
+              {{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }}
```

Comment:
```suggestion
              <i>{{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }}</i>
```
Gene symbols in italics whenever reasonably possible.